README.md

```diff
@@ -1,118 +1,80 @@
-# Jenkins Folder Properties Plugin
+A Jenkins plugin which allows users with config permission to define
+properties for a folder which can then be used by any jobs contained
+within it or any of its sub-folders. 
 
-A Jenkins plugin which allows users with config permission to define properties for a folder which can then be used by
-any jobs contained within it or any of its sub-folders.
+You can find [this plugin's source code on
+Github](https://github.com/jenkinsci/folder-properties-plugin).
 
-The aim here is to remove the need to specify the same properties over and over again for all the jobs inside a folder.
+ 
 
-In structures where two or more folders are nested, any property defined for a folder will be overridden by any other property of the same name defined by one of its sub-folders.
+# About this Plugin
 
-## Develop
+The aim here is to remove the need to specify the same properties over
+and over again for all the jobs inside a folder.
 
-For ease of development I've added a `logging.properties` file that can be used by specifying it as an option to the
-Maven `hpi:run` target like so:
+In structures where two or more folders are nested, any property defined
+for a folder will be overridden by any other property of the same name
+defined by one of its sub-folders.
 
-    mvn hpi:run -Djava.util.logging.config.file=logging.properties
+## Configuring Folder Properties
 
-## Package
+Just create a
+[folder](http://localhost:8085/display/JENKINS/CloudBees+Folders+Plugin),
+go to its configuration page and add as many properties as you need
+under the `Folder Properties` section.
 
-Since this is my first version of this and I'm still missing the tests...
+![](docs/images/folder-properties-config.png){height="250"}
```

Comment:
```suggestion
![](docs/images/folder-properties-config.png)
```
This property is visible in the file, do we need to add this?

Reply:
Yes, I guess this property is needed. And I didn't get what you meant by "_visible_" in file. Can you explain more?

Reply:
Sure, so if you go to your copy of the README.md file, you'd be able to see "{height="250"}" displayed just below your image. We might want to remove it from there.

Reply:
Yup, understood. Thanks for pointing it out @dheerajodha !!. I'll fix it asap.